docs/en/ingest-management/fleet/fleet-server-monitoring.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server-monitoring]]
+= {fleet-server} monitoring
+
+Monitoring {fleet-server} is key since the operation of the {fleet-server} is
+paramount to the health of the deployed agents and the services they offer. When
+{fleet-server} is not operating correctly, it may lead to delayed check-ins,
+status information, and updates for the agents it manages. The monitoring data
+will tell you when to add capacity for {fleet-server}, and provide error logs
+and information to troubleshoot other issues.
+
+To enable monitoring for {fleet-server}, turn on agent monitoring in the agent
+policy. For self-managed clusters, monitoring is on by default when you create a
+new agent policy or use the existing Default {fleet-server} agent policy.
+However, it is off by default in the {ecloud} agent policy because monitoring
+requires additional RAM.
+
+To turn on {fleet-server} monitoring in the agent policy:
```

Comment:
@dedemorton I believe the docs are wrong in that we don't allow this.  The UI might have *seemed to let you do it, but we've recently fixed that I think.  QA logged a bug about it and was confirmed, see notes in this issue:
https://github.com/elastic/kibana/issues/110533#issuecomment-908686621
and this is the 'question' bug that QA logged: https://github.com/elastic/kibana/issues/117809

Reply:
This isn't new content; it's just moved from the long Fleet Server topic. So it has not been tested recently.

@nimarezainia Can you let me know how to proceed here? I can remove the content, but I don't want to make the call without your input.

Reply:
@EricDavisX Actually, can you [open an issue](https://github.com/elastic/observability-docs/issues/new/choose) about this in the observability-docs repo? I don't want this PR to get stalled since the changes are structural, not content.